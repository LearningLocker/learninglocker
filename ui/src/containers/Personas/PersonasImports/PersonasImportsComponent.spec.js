import React from 'react';
import { shallow } from 'enzyme';
import 'jest-enzyme';
import { PersonasImportsComponent } from './index';

describe('PersonasImports with items', () => {
  it('should render', () => {
    const rendered = shallow(
      <PersonasImportsComponent
        onAddModel={jest.fn()} />
    );

    expect(rendered).toMatchSnapshot();
  });
});
