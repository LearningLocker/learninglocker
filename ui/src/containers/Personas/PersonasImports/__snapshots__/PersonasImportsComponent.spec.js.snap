// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonasImports with items should render 1`] = `
<div>
  <header
    id="topbar"
  >
    <div
      className="heading heading-light"
    >
      <span
        className="pull-right open_panel_btn"
      >
        <button
          className="btn btn-primary btn-sm"
          onClick={[Function]}
        >
          <i
            className="ion ion-plus"
          />
          Import
        </button>
      </span>
      People - Imports
    </div>
  </header>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <withProps(Connect(WithModels))
        ModelForm={[Function]}
        filter={Object {}}
        getDescription={[Function]}
      />
    </div>
  </div>
</div>
`;
